### DISCUSSION TITLE
Reveal JS : How to set text color for a specific slide?

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#1913] [Q&A]
I have slides with custom background. How to also set a specific color text for this slides ?

**Comment by jjallaire** [comment]
You can set the `background-color` explicitly to let Reveal know that it should invert the normal text color (e.g. use white text b/c the background color is dark). See https://quarto.org/docs/presentations/revealjs/index.html#slide-backgrounds
